controllers/collection_exercise_controller.py

```diff
@@ -269,10 +269,11 @@ def create_business_collection_exercise_to_ready_for_review_state(survey_id, per
 
     if eq_ci:
         ci_controller.load_and_link_eq_collection_instrument(survey_id, collection_exercise_id,
-                                                             form_type=ru_ref, eq_id=ru_ref)
+                                                             form_type='0001', eq_id=ru_ref)
```

Comment:
Is there a reason these are hardcoded to `0001`? Could be worth a comment to avoid accidental changing and bugs in the future